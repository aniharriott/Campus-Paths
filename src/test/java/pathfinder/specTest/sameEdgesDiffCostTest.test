# Test that findMinPath will return least cost path

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2

AddEdge graph1 n1 n2 6.000
AddEdge graph1 n1 n2 16.000

FindPath graph1 n1 n2