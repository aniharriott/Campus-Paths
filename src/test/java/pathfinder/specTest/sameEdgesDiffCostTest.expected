# Test that findMinPath will return least cost path

created graph graph1

added node n1 to graph1
added node n2 to graph1

added edge 6.000 from n1 to n2 in graph1
added edge 16.000 from n1 to n2 in graph1

path from n1 to n2:
n1 to n2 with weight 6.000
total cost: 6.000